server/src/main/java/keywhiz/service/crypto/RowHmacGenerator.java

```diff
@@ -35,6 +35,11 @@ public String computeRowHmac(String table, long id1, long id2) {
     return cryptographer.computeHmacWithSecretKey(hmacContent.getBytes(UTF_8), hmacKey);
   }
 
+  public String computeRowHmac(String table, String content, String metadata, long id) {
```

Comment:
copy pasta with DbSeedCommand.java. DbSeedCommand.java should just call this version.